lib/web_ui/dev/test_runner.dart

```diff
@@ -46,9 +57,14 @@ class TestCommand extends Command<bool> {
   @override
   final String description = 'Run tests.';
 
+  // Browsers supported by this platform.
+  final SupportedBrowsers supportedBrowsers = SupportedBrowsers();
+
   @override
   Future<bool> run() async {
-    Chrome.version = chromeVersion;
+    assert(supportedBrowsers.supportedBrowserNames.contains(browser));
```

Comment:
ditto: assertions don't work in the tool

Reply:
I think there were some in Chrome (browser class) before my changes. Shall I remove all the asserts from the felt code?

Reply:
done (for test_runner and test_platform)

Reply:
:+1: